zipkin-api.yaml

```diff
@@ -0,0 +1,259 @@
+swagger: "2.0"
+info:
+  version: "1.0.0"
+  title: Zipkin API
+  description: |
+    Zipkin's Query api is rooted at `api/v1`, on a host that by default listens
+    on port 9411. It primarily serves zipkin-web, although it includes a POST
+    endpoint that can receive spans.
+host: localhost:9411
+basePath: /api/v1
+schemes:
+  - http
+  - https
+consumes:
```

Comment:
there's only one "consumes" aka POST endpoint. that also accepts application/x-thrift